api/overview/azure/batch.md

```diff
@@ -29,12 +37,44 @@ Install-Package Azure.Batch
 dotnet add package Azure.Batch

```

Comment:
Why do we label the .NET CLI sections as bash? The command is the same for non-bash environments.